pkg/clients/clients.go

```diff
@@ -196,6 +198,33 @@ func (k *k8sClients) GetPlatform() (string, error) {
 	}
 }
 
+func (k *k8sClients) GetNodesByLabels(ctx context.Context, matchingLabels map[string]string) (*v1.NodeList, error) {
+	nodes, err := k.clientset.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: labels.Set(matchingLabels).String()})
```

Comment:
Shall we use controller-runtime's `client.Client` interface interface instead? I would try to standardize on it as much as possible (we only keep a ClientSet in SRO because we need to read pod logs).

Reply:
actually CoreV1 is a more accepted way to access basic K8s  structures (namespaces, pods, nodes,etc').  we use in in ClientsInterface to access not only pods logs, but also namespaces and secrets

Reply:
I think they're really equivalent; I personally don't have a real preference, but we use `client.Client` more than the regular `ClientSet`. Why not standardize on one?